applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -317,6 +317,9 @@ The update image can also be transferred in the background through one of the fo
 
 The `nRF Connect Device Manager`_ application transfers the image update files over the Simple Management Protocol (SMP).
 
+.. note::
+   If your DFU target is not paired over Bluetooth with your Android device, the DFU procedure will automatically trigger the pairing procedure.
```

Comment:
Consider mentioning explicitly that encryption is required for SMP over BLE by nRF Desktop configurations (I am not sure if other host tools, like mcumgr CLI, would also automatically establish security).